transformer/parse.go

```diff
@@ -0,0 +1,123 @@
+package transformer
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/telenornms/skogul"
+	"github.com/telenornms/skogul/parser"
+)
+
+var parseLog = skogul.Logger("transformer", "parser")
+
+type Parse struct {
+	Parser              string `doc:"Name of skogul parser to use. Note: has to be auto-initialisable."` // Feature request: Re-use parser unmarshal logic here
```

Comment:
This should be a ParserRef instead of a string. That will make the config engine resolve the parser for you making that part of the PR redundant. With the implicitly created modules, it would still work just fine without requiring the user to specify a json-parser, but it will allow them to use parsers that have configuration in the future.

The only change should be changing this from string to ParserRef, removing the lower-case "parser", removing the lookup-code, and using Parser.P instead of lowercase "parser".